pages/specification-links.md

```diff
@@ -301,8 +299,8 @@ For links to the somewhat more readably formatted versions on this web site, and
 ### 2020-12
 
 - Specifications
-    - Core: [draft-bhutton-json-schema-01](https://tools.ietf.org/html/draft-bhutton-json-schema-01) ([changes](https://tools.ietf.org/html/draft-bhutton-json-schema-01#appendix-G))
-    - Validation: [draft-bhutton-json-schema-validation-01](https://tools.ietf.org/html/draft-bhutton-json-schema-validation-01) ([changes](https://tools.ietf.org/html/draft-bhutton-json-schema-validation-01#appendix-C))
+    - Core: [draft-bhutton-json-schema-01](/draft/2020-12/draft-bhutton-json-schema-01.html) ([changes](/draft/2020-12/draft-bhutton-json-schema-01.html#appendix-G))
+    - Validation: [draft-bhutton-json-schema-validation-01](/draft/2020-12/draft-bhutton-json-schema-validation-01.html) ([changes](/draft/2020-12/draft-bhutton-json-schema-validation-01.html#appendix-C))
```

Comment:
Why are the other entries split with "Core:" on one line and the links on another, but this one is not?  (I think I prefer not split.)

Reply:
I think they're all supposed to be on the same line. There seems to be a mix. I'll normalize them.